src/test/scala/io/confluent/examples/streams/ProbabilisticCountingScalaIntegrationTest.scala

```diff
@@ -82,13 +71,9 @@ class ProbabilisticCountingScalaIntegrationTest extends AssertionsForJUnit {
     val streamsConfiguration: Properties = {
       val p = new Properties()
       p.put(StreamsConfig.APPLICATION_ID_CONFIG, "probabilistic-counting-scala-integration-test")
-      p.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, cluster.bootstrapServers())
+      p.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy config")
       p.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.ByteArray.getClass.getName)
```

Comment:
@vvcephei : Do we even need to set the default serdes anymore?  In the case of the Scala API for KStreams, the API should automatically pick up the right serdes via Scala implicits (at least for the primitive types like String etc., which are all supported OOTB by KStreams in Scala).

Reply:
Unsure! I didn't think about that. I'll try it out.

Reply:
Ah, these examples haven't been ported to the actual scala API yet. I think I'll defer that work till a later PR, to keep this one nice and formulaic.